src/docs/Welcome.mdx

```diff
@@ -7,7 +7,75 @@ import Icon from "../components/Icons/Icon";
 import Link from "../components/Link/Link";
 import Logo from "../components/Logo/Logo";
 import SimpleGrid from "../components/Grid/SimpleGrid";
+import Table from "../components/Table/Table";
 import Text from "../components/Text/Text";
+export const versionsHeadersData = [
+  "Version",
+  "Date",
+  "Summary",
+  "Documentation",
+  "Code",
+];
+export const versionsData = [
+  [
+    <Text size="body2" whiteSpace="nowrap">
+      Reservoir 2.1.6
+    </Text>,
+    <Text size="body2" whiteSpace="nowrap">
+      September 2023
+    </Text>,
+    <Text size="body2">
+      Reservoir v2 added significant typography style changes. Reservoir v2.1.6
+      is the last release in the v2 series.
+    </Text>,
+    <Text size="body2" whiteSpace="nowrap">
+      <Link
+        href="https://nypl-design-system-git-reservoir-v216-nypl.vercel.app"
+        fontSize="desktop.body.body2"
+        type="external"
+      >
+        v2.1.6 in Storybook
+      </Link>
+    </Text>,
+    <Text size="body2" whiteSpace="nowrap">
+      <Link
+        href="https://github.com/NYPL/nypl-design-system/tree/reservoir-v2.1.6"
```

Comment:
NBD but I wonder if pointing to the tag might be better than the branch. This is fine but maybe we can offer or point to both?

Reply:
Hmm, yes, that is a good question.  The tag would get a user to the downloadables, but is tha tmore important than the branch in the repo itself?

Reply:
This would work...
![image](https://github.com/NYPL/nypl-design-system/assets/5241828/f99b1b53-53be-4ee1-ad62-24959269bbbb)

Reply:
Hmm either way, if someone wants the 2.1.6 version, they would install it through npm. But I like the idea of pointing to the branch/tag in case someone wants to see the code.